src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java

```diff
@@ -0,0 +1,92 @@
+package org.folio.circulation.domain.policy.lostitem;
+
+import static org.folio.circulation.domain.policy.Period.from;
+import static org.folio.circulation.domain.policy.Period.minutes;
+import static org.joda.time.DateTime.now;
+import static org.joda.time.DateTimeZone.UTC;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Random;
+
+import org.folio.circulation.domain.policy.Period;
+import org.joda.time.DateTime;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.vertx.core.json.JsonObject;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public class LostItemPolicyTest {
```

Comment:
Why is it important that the duration of the aged to lost period is random within these tests?

Reply:
It is not important, it can be constant.

Reply:
They are constants now.